game-03/README.md

```diff
@@ -0,0 +1,37 @@
+# Godot Minigames
```

Comment:
Can you remove the docs from the moddable platformer? Instead use this template:
```
# [game title]

## About
[What is this game about? Describe the goal and the challanges.]

## Implementation
[Please describe how you managed to implement this without a single line of GDScript. Which blocks were useful? Which editor features and nodes? Have you found interesting ways of using the current blocks? Have you been surprised by something that required no programming at all in Godot?]

## Limitations found
[What had to be workarounded and should be easier? What wasn't possible? Which blocks were you missing?]

```

Reply:
sure.  Delete it first.  Will add the README.md back after the game development reaches a milestone.